docs/UserGuide.md

```diff
@@ -62,111 +72,141 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
+**:information_source: Notes about the application:**<br>
+- For **ALL** entries, categories are <ins>COMPULSORY</ins> and every entry can only contain <ins>one</ins> category. 
+  The following table shows the`Expenditure` and `Income` entries categories. The category names are <ins>case sensitive</ins>.
+
+|   `Expenditure`   |    `Income`     |
+|:-----------------:|:---------------:|
+|     **Food**      |   **Salary**    |
+|   **Groceries**   |  **Allowance**  |
+| **Entertainment** |   **Profit**    |
+|   **Education**   | **Investments** |
+|    **Housing**    |    **Gifts**    |
+|    **Others**     |   **Others**    |
+
+- For **ALL** entries, 2 entries are considered duplicates <ins>IF</ins> both entries have exactly the same:
+  `description`, `date` `amount` and `category`. We do not want PennyWise to be managing entries which are a repeat,
+  of one another, as it would be easier to simply use the [edit](#editing-entries--edit-index_of_entry-tentry_type-dedited_description-aedited_amount-daedited_date-cedited_category) 
+  command to alter the original entry's specifications.
+  e.g. Executing the following 2 commands one after another, PennyWise will recognise (2) as a duplicate entry. 
+  <pre>
+    <code>
+        1. add t/e d/Teh Beng a/1.50 da/22-10-2022 c/Food
+        2. add t/e d/Teh Beng a/1.50 da/22-10-2022 c/Food
+    </code>
+  </pre>
+  To add similar entries, vary the description/amount/date/category to let PennyWise know they are not duplicates!
+  <pre>
+    <code>
+        1. add t/e d/Teh Beng 11am a/1.50 da/22-10-2022 c/Food
+        2. add t/e d/Teh Beng 4pm a/1.50 da/22-10-2022 c/Food 
+    </code>
+  </pre>
 
-Adds an income entry.
+</div>
 
-1. Add income with default current date
-   * Example: `add -i 15.60`
-   * An income of $15.60 will be added with the current date
-   * Expected: `An income of 15.60 on 27 Sep 2022 has been recorded`
-2. Add income with specific date
-   * Example: `add -i 15.60 / 12-09-2022`
-   * An income of $15.60 will be added with 12 September 2022 as the date
-   * Expected: `An income of 15.60 on 12 Sep 2022 has been recorded`
+### Adding entries : `add t/ENTRY_TYPE d/DESCRIPTION a/AMOUNT da/DATE c/CATEGORY`
+Adds an entry to the specified list. <br>
+1. Add expense <br>
+   - Example: `add t/e d/Lunch a/15.60 da/10-10-2022 c/Food`<br>
+   - An expense of 15.60 on 10/10/2022 will be added to the Expenditure list, under the Food category.
+
+1. Add income <br>
+   - Example: `add t/i d/Tuition a/40.00 da/10-10-2022 c/Salary`<br>
+   - An income of 40.00 on 10/10/2022 will be added to the Income list, under the Salary category.
+
+### Deleting entries : `del INDEX_OF_ENTRY t/ENTRY_TYPE`
+Deletes an entry.
+1. Example: `del 2 t/e` deletes the 2nd item on the expenditure list 
+   Expenditure list:
+   1. Movie 15.60 12 Sep 2022 c/Entertainment
+   2. Drink 1.20 12 Sep 2022 c/Food 
+   Expected: `Deleted Entry: Drink; Date: 12-09-2022; Amount: 1.20; Tag: Food`
```

Comment:
When I viewed the diff the Expected line appears on the same line as the "2. `Drink 1.20  12 Sep 2022 c/Food` line". Not sure if its just GitHub or a bug

Reply:
have added a `<br>` in between all the expected outcomes to fix this